### DISCUSSION TITLE
[type_caster<char>] Support of nullptr from cpp?

### DISCUSSION THREAD

**Discussion by huangweiwu** [wjakob/nanobind#296] [Q&A] [answered]
When I bind a const char*<> type function, it will crash when returns a nullptr.
Shell we support it in the from_cpp function of [type_caster<char>]?

```
const char* test_func()
{
    return nullptr;
}
m.def("test_func", test_func);
```

**Comment by wjakob** [comment]
Sorry, I saw this just now. That would be reasonable. Could you make a PR?

**Reply by huangweiwu** [reply]
I have no permission to push to this project currently.

**Reply by wjakob** [reply]
The way to do it is to create a fork, make changes in a branch, and then open a pull request. If those terms are unclear to you, then please review the documentation of GitHub (I cannot provide that level of support here.)

**Reply by huangweiwu** [reply]
It's ok now.

**Reply by wjakob** [reply, marked as answer]
fixed by @huangweiwu in PR #318.
